lib/ansible/modules/cloud/amazon/s3_bucket.py

```diff
@@ -555,11 +636,15 @@ def main():
             state=dict(default='present', type='str', choices=['present', 'absent']),
             tags=dict(required=False, default=None, type='dict'),
             versioning=dict(default=None, type='bool'),
-            ceph=dict(default='no', type='bool')
+            ceph=dict(default='no', type='bool'),
+            encryption=dict(choices=['none', 'AES256', 'aws:kms']),
+            encryption_key_id=dict()
         )
     )
 
-    module = AnsibleAWSModule(argument_spec=argument_spec)
+    module = AnsibleAWSModule(
+        argument_spec=argument_spec, required_if=[['encryption', 'aws:kms', ['encryption_key_id']]]
```

Comment:
```suggestion
        argument_spec=argument_spec, 
        required_if=[['encryption', 'aws:kms', ['encryption_key_id']]]
```
The line break should separate the args, see the style of other aws modules for examples.